.github/workflows/integration.yml

```diff
@@ -302,12 +305,19 @@ jobs:
         with:
           name: a3p-integration
           path: /tmp/export/a3p-integration
-      - name: archive slogfile
+      - name: archive build slogfile
+        if: (success() || failure())
+        uses: actions/upload-artifact@v4
+        with:
+          name: build slogfile
+          path: ${{ format('{0}/build.slog', env.HOME) }}
+          retention-days: 10
+      - name: archive test slogfile
         if: (success() || failure())
         uses: actions/upload-artifact@v4
         with:
-          name: slogfile
-          path: ${{ env.SLOGFILE }}
+          name: test slogfile
```

Comment:
Almost certainly not since its presence is so recent, but I'll be reverting this change anyway because it's not a viable means of extracting slogfiles from `docker build`.